### PR TITLE
changed border shadow, modified transition, added a pill scss variable

### LINKED ISSUE #212: Update button styles
Primary and Secondary button hover/focus states need updating. I am removing the bottom border darkening and adding a full on background-shade.

Please add a transition of .1s linear on the background color so it flows smoothly.

Additionally, I've added styles for Primary and Secondary round buttons.  There are instances where we'll need round buttons so please add them to the component :)

image

### PR DESCRIPTION
**Pull request by splinterific**
# Details

## Description

Updated button styles for background colour, gradient, transition and style for rounded button/pill button.

Closes #212

### CODE REVIEW

File: src/components/button/Button.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Button = (props) => {
         styles[typography] || styles.uppercase,
       )}
     >
-      {children}
+      {children}{' '}
```
Comment (crysfel) [inline src/components/button/Button.js]: Is the extra `{' '}` necessary?

Comment (splinterific) [reply src/components/button/Button.js]: ugh, thats my linter doing that. :( Want me to remove it and resubmit?

Comment (crysfel) [reply src/components/button/Button.js]: yes please

Comment (tureey) [reply src/components/button/Button.js]: +1. Gave the approval but please, remove this! Good job :+1:

**Comment by emmawedekind** [comment]
Looks great Mark!

File: src/components/button/Button.js (hunk 1 above)
Comment (splinterific) [reply src/components/button/Button.js]: removed :)

**Pull request merged**